test/extended/testdata/builds/volumes/buildconfig-docker.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: List
+metadata: {}
+items:
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: mydockertest
+    labels:
+      name: test
+  spec:
+    triggers: []
+    runPolicy: Serial
+    source:
+      dockerfile: |
+        FROM image-registry.openshift-image-registry.svc:5000/openshift/tools:latest
+    strategy:
+      type: Docker
+      dockerStrategy:
+        env:
+          - name: BUILD_LOGLEVEL
+            value: "5"
+    resources: {}
+    postCommit: {}
+    nodeSelector: null
+  status:
+    lastVersion: 0
```

Status should be empty 
```suggestion
  status: {}

```

---

updated in next push